CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [next-version]
 
+### Fixed:
+
+- Public: Fix issue where multiple SDK instances were sharing the same Redux store values. This was resulting in duplicate cross-device links across multiple instances. With this change, the redux store is reset every time one SDK instance is unmounted via the `tearDown()` function.
```

Comment:
> multiple SDK instances were sharing the same Redux store values

Can we have more details on this use case? More importantly, why are we accepting this kind of use case?

Reply:
Some clients want to capture poa+idv or capture multiple documents as part of the same flow. We know that there have been issues when trying to initialise the SDK with multiple document steps or poa+idv, so these clients have been advised by product support to initialise multiple the SDK flows, where every time the onComplete callback is fired they re initialise the SDK.
It looks something like:
```
 window.onfidoOut = init({
      token,
      onComplete: function (data) {
        const firstFlowData = data
        console.log('first flow completed, initialising second flow')
        // callback for when everything is complete
        window.onfidoOut?.tearDown()
        init({
          onComplete: function (secondFlowData) {
            console.log(
              'Both SDK flows have been completed with the following data',
              { firstFlowData, secondFlowData }
            )
          },
          token,
          steps: [
            "document",
            "complete"
          ],
        })
      },
      steps: ["document"],
    })

Reply:
Wow nested `init`, that's evil :trollface: